src/crypto/digibyte.ts

```diff
@@ -11,6 +13,19 @@ class Digibyte extends Base {
     this.network.dustPolicy = 'FIXED';
     this.network.usesTimestampedTransaction = false;
   }
+
+  validateAddress(address: string): boolean {
+    // bech32 address
+    if (address.substring(0, 3) === 'dgb') {
+      if (bech32.decodeUnsafe(address) || bech32m.decodeUnsafe(address)) {
```

Comment:
Using decodeUnsafe is a nice touch here, so that we don't have to `try...catch` everything.

Just making sure: Do digibyte actually use bech32m?

Reply:
good point. I've removed bechm for litecoin and digibyte